src/System Application/App/Entity Text/src/EntityTextCapability.EnumExt.al

```diff
@@ -3,6 +3,10 @@ using System.AI;
 
 enumextension 2015 "Entity Text Capability" extends "Copilot Capability"
 {
+    value(2014; "Reminder Text")
```

Comment:
I don't think that it make sense to have this value here, as the "Entity Text" app should be agnostic to the different implementations - and in this situation you are adding some feature specific code to a generic tool.